llvm/lib/Target/WebAssembly/WebAssemblyCFGStackify.cpp

```diff
@@ -77,8 +78,8 @@ class WebAssemblyCFGStackify final : public MachineFunctionPass {
   // Exception handling related functions
   bool fixCallUnwindMismatches(MachineFunction &MF);
   bool fixCatchUnwindMismatches(MachineFunction &MF);
-  void addTryDelegate(MachineInstr *RangeBegin, MachineInstr *RangeEnd,
-                      MachineBasicBlock *DelegateDest);
+  void addNestedTryDelegate(MachineInstr *RangeBegin, MachineInstr *RangeEnd,
```

Comment:
We didn't really need 'nested' here because `try`-`delegate` is only used for fixing unwind mismatches, but now for the new EH we need to use `try_table` to fix mismatches, and that function's name is going to be `addNestedTryTable` (because `try_table` can be used for normal markers too). This is to make this function rhyme with the new one.